docs/3.0.0-beta.x/getting-started/install-requirements.md

```diff
@@ -1,127 +1,27 @@
 
 This is all that is needed before Strapi can run on your local environment.
 
-## Installing Node.js
-
-You can install `Node.js` and `npm` for `Windows 10`, `Ubuntu 18.04` and `Mac O/S Mojave` with the following instructions.
-
-#### Installation Instructions for each Operating System:
-
-:::: tabs
-
-::: tab "Windows 10"
-
-### Installing Node.js on Windows 10
-
-There are several methods to install Node.js on _Windows 10_.
-
-We will follow the most common download and installation procedure. These _instructions are for Windows 10_. (If you are installing on a different version of Windows or if you have trouble following these instructions, please review the [official Node.js documentation](https://nodejs.org/en/docs/).)
-
-1. Download the Windows Installer from the [downloads page](https://nodejs.org/en/download/). You will need to choose the 32-bit or 64-bit version. We recommend the LTS (long-term support) version of Node.js.
-2. Double-click the node-v10.x.x-x86.msi file icon. Click "Next" for the default options and to install Node.js under the recommended and most common settings. After clicking "Next" several times, click "Install" to install Node.js. When it is done installing, click "Finish".
-3. Verify both Node.js and npm have installed correctly. Open your Command Prompt:
-   - Click your Start Button
-   - In Search type, "cmd"
-   - Then click on "Command Prompt".
-   - Type the following commands in your Command Prompt
-
-Verify Node.js has correctly installed:
-
-```shell
-node -v
-## You should see "v10.x.x
-```
-
-Next, verify npm has correctly installed:
-
-```shell
-npm -v
-## You should see "6.x.x"
-```
-
-:::
-
-::: tab "Mac O/S"
-
-### Installing Node.js on Mac O/S X (Mojave)
-
-There are multiple methods to install Node.js on _Mac O/S X (Mojave)_.
-
-We will follow the most common download and installation procedure. _These instructions are for Mac O/S X (Mojave)_. (If you are installing on a different version of Mac O/S or if you have trouble following these instructions, please review the [official Node.js documentation](https://nodejs.org/en/docs/).)
+You can also use **yarn** if you want [here](https://yarnpkg.com/en/docs/getting-started) are the instructions to get started with it.
 
-1. Download the Mac O/S Installer from the [downloads page](https://nodejs.org/en/download/). We recommend the LTS (long-term support) version of Node.js.
-2. Launch the node-v10.x.x.pkg file icon. Click "Continue" for the default options and to install Node.js under the recommended and most common settings. After clicking "Continue" several times, click "Install" to install Node.js. When it is done installing, click "Close".
-3. Verify both Node.js and npm have installed correctly. Open your terminal prompt:
-   - Open your Applications folder
-   - Find and open the Utilities folder
-   - Find "Terminal" and click on it to open it.
-   - Type the following commands in your Terminal
+- **Node.js** >= 10.x
```

Comment:
We need to change this to node 12, node 10 is no longer LTS (but this might be something we need to do documentation wide, not just here)